### ISSUE TITLE
Missing SFNode import/remove functionality in supervisor API

### ISSUE THREAD

**Issue by DavidMansolino** [cyberbotics/webots#748]
The Superivisor APi is missing the following functions:
```
void wb_supervisor_field_remove_sf(WbFieldRef field);
void wb_supervisor_field_import_sf_node(WbFieldRef field, const char *filename);
void wb_supervisor_field_import_sf_node_from_string(WbFieldRef field, const char *node_string);
```

**Comment by stefaniapedrazzi** [comment]
Just for note here is an old issue related to this:
https://github.com/omichel/webots-dev/issues/5589

**Comment by DavidMansolino** [comment]
Fixed in #1145.
